### ISSUE TITLE
ofxCardboard for iOS

### ISSUE THREAD

**Issue by julapy** [danthemellowman/ofxCardboard#2]
hey Dan,

this is not an issue, just chats...

im looking into getting cardboard running with ofxiOS.
how far did you get with your implementation?

ive been looking at https://github.com/rsanchezsaez/CardboardSDK-iOS
and the example is compiling fine, so thinking the next step will be to try to get it running with ofxiOS

let me know if you have any thoughts / recommendations.

cheers,

**Comment by danthemellowman** [comment]
Yeah I did a bunch of work then accidentally blew away most of my android
core changes to make this work unless i stashed this somewhere. I need to
report the newer android Lib and get the actual shaders written

On Wednesday, December 2, 2015, lukasz karluk <notifications@github.com>
wrote:

> hey Dan,
>
> this is not an issue, just chats...
>
> im looking into getting cardboard running with ofxiOS.
> how far did you get with your implementation?
>
> ive been looking at https://github.com/rsanchezsaez/CardboardSDK-iOS
> and the example is compiling fine, so thinking the next step will be to
> try to get it running with ofxiOS
>
> let me know if you have any thoughts / recommendations.
>
> cheers,
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/danthemellowman/ofxCardboard/issues/2>.
>


-- 
Sent from Gmail Mobile #autocorrect

**Comment by jamezilla** [comment]
@julapy @danzeeeman did either of you get further with these projects? I'm looking for an iOS panoramic video player suitable for use with a cardboard-style viewer.
